src/facade/dragonfly_connection.cc

```diff
@@ -48,7 +48,7 @@ ABSL_FLAG(uint64_t, pipeline_queue_limit, 1ULL << 27,  // 128MB
 
 ABSL_FLAG(bool, no_tls_on_admin_port, false, "Allow non-tls connections on admin port");
 
-ABSL_FLAG(uint64_t, pipeline_squash, 0,
+ABSL_FLAG(uint64_t, pipeline_squash, 20,
```

Comment:
my reservation is that this codepath was not widely tested and by setting a high bar now we may allow bugs that will be rare. Why not enable low thresholds - so at least we will have some mileage in regression tests, community maybe to gain higher confidence in this feature?